packages/react/src/patterns/blocks/ContentGroupHorizontal/README.md

```diff
@@ -0,0 +1,100 @@
+@include carbon--font-face-sans();
+
+@import '@carbon/ibmdotcom-styles/scss/patterns/sub-patterns/content-group-horizontal/content-group-horizontal';
+```
+
+> 💡 Only import fonts once per usage. Don't forget to import the
+> content-group-horizontal styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+##### JS
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { ContentGroupHorizontal } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  const items = [
+    {
+      eyebrow: 'Lorem ipsum',
+      heading: 'Aliquam condimentum',
+      copy:
+        'Lorem ipsum dolor sit amet, _consectetur_ sellus at elit sollicitudin.',
+      cta: [
+        {
+          type: 'local',
+          copy: 'Link text',
+          href: 'https://example.com',
+        },
+        {
+          type: 'external',
+          copy: 'External link text',
+          href: 'https://example.com',
+        },
+      ],
+    },
+    {
+      eyebrow: 'Lorem ipsum',
+      heading: 'Aliquam condimentum',
+      copy:
+        'Lorem ipsum dolor sit amet, _consectetur_ adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin.',
+    },
+  ];
+
+  return <ContentGroupHorizontal heading="Aliquam condimentum" items={items} />;
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+Add the following line in your `.env` file at the root of your project.
+[See more details](https://github.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/styles#usage)..
+
+```
+  SASS_PATH=node_modules:src
+```
+
+## Props
+
+| Name      | Required | Data Type | Default Value | Description                                                                                                                                                                                                   |
+| --------- | -------- | --------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `heading` | YES      | String    | null          | Heading of the content group.                                                                                                                                                                                 |
+| `items`   | YES      | Array     | null          | Array of content items. See [ContentItemHorizontal](https://github.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/react/src/patterns/sub-patterns/ContentItemHorizontal) for usage details. |
+
+## Stable selectors
+
+| Name                              | Description |
+| --------------------------------- | ----------- |
+| `dds----content-group-horizontal` | Pattern     |
```

Comment:
@kennylam minor thing, looks like too many dashes in this stable selector name.

Reply:
@jeffchew I feel like I do this on every PR :(